NinjaNye.SearchExtensions/QueryableStringSearch.cs

```diff
@@ -16,6 +16,7 @@ namespace NinjaNye.SearchExtensions
     public class QueryableStringSearch<T> : QueryableSearchBase<T, string>
     {
         private IList<string> _containingSearchTerms = new List<string>();
+        private IList<string> _startsWithSearchTerms = new List<string>();
```

Comment:
I think we can simply add the `StartsWith` search terms to the `_containingSearchTerms` variable meaning we can remove the new `_startsWithSearchTerms` completely